[2025-01-29T16:22:28.557+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T16:22:28.589+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.upload_s3 manual__2025-01-29T16:22:08.198012+00:00 [queued]>
[2025-01-29T16:22:28.604+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.upload_s3 manual__2025-01-29T16:22:08.198012+00:00 [queued]>
[2025-01-29T16:22:28.605+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2025-01-29T16:22:28.631+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): upload_s3> on 2025-01-29 16:22:08.198012+00:00
[2025-01-29T16:22:28.671+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2059) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T16:22:28.677+0000] {standard_task_runner.py:72} INFO - Started process 2061 to run task
[2025-01-29T16:22:28.678+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'upload_s3', 'manual__2025-01-29T16:22:08.198012+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp_1xmxzje']
[2025-01-29T16:22:28.686+0000] {standard_task_runner.py:105} INFO - Job 90: Subtask upload_s3
[2025-01-29T16:22:28.804+0000] {task_command.py:467} INFO - Running <TaskInstance: main_dag.upload_s3 manual__2025-01-29T16:22:08.198012+00:00 [running]> on host ae52aba2dca2
[2025-01-29T16:22:29.071+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='itsMe!!' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='upload_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T16:22:08.198012+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T16:22:08.198012+00:00'
[2025-01-29T16:22:29.074+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T16:22:35.750+0000] {logging_mixin.py:190} INFO - !_S3_upload_complete_onlineII.csv!
[2025-01-29T16:22:35.754+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-29T16:22:35.779+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T16:22:35.781+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=upload_s3, run_id=manual__2025-01-29T16:22:08.198012+00:00, execution_date=20250129T162208, start_date=20250129T162228, end_date=20250129T162235
[2025-01-29T16:22:35.871+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T16:22:35.948+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T16:22:35.952+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
